src/core/namespace.c

```diff
@@ -319,8 +327,11 @@ static int apply_mount(
         r = mount(what, m->path, NULL, MS_BIND|MS_REC, NULL);
         if (r >= 0)
                 log_debug("Successfully mounted %s to %s", what, m->path);
-        else if (m->ignore && errno == ENOENT)
-                return 0;
+        else {
+                if (m->ignore && errno == ENOENT)
+                        return 0;
+                log_error_errno(errno, "Failed mounting %s to %s: %s", what, m->path, strerror(errno));
```

Comment:
don't log here at anything but debug level. let the caller log. Hence, either drop the logging here, or downgrade it to debug.

@evverx doing debug logging here is fine, even if it has no effect when this code is used by PID 1. Why? because there's src/tests/test-ns.c, where logging works correctly and highly desirable.